### PR TITLE
refactor(ripple): improve ripple samples

### PR DESCRIPTION
**Pull request by SisIvanova**
Closes: #1875

### CODE REVIEW

File: src/app/interactions/ripple/ripple-sample-3/ripple-sample-3.component.scss [hunk 1]
```diff
@@ -1,3 +1,4 @@
 .sample-button{
     width: 7rem;
-}
\ No newline at end of file
+    margin: 10px;
```
Comment (simeonoff) [inline src/app/interactions/ripple/ripple-sample-3/ripple-sample-3.component.scss]: Make sure that paddings and margins are divisible by 8. We base our designs on a 8x8 grid.
```suggestion
    margin: 8px;
```

File: src/app/interactions/ripple/ripple-sample-4/ripple-sample-4.component.html [hunk 2]
```diff
@@ -1,3 +1 @@
-<div class="ripple-sample dark" [igxRippleDuration]=2000 igxRipple="white">
-  <p style="margin-bottom:0">Long Ripple Animation</p>
-</div>
\ No newline at end of file
+<button class="sample-button" igxButton="raised" igxRipple [igxRippleDuration]=2000>Click Me</button>
```
Comment (simeonoff) [inline src/app/interactions/ripple/ripple-sample-4/ripple-sample-4.component.html]:
```suggestion
<button class="sample-button" igxButton="raised" igxRipple igxRippleDuration="2000">Click Me</button>
```

File: src/app/interactions/ripple/ripple-sample-4/ripple-sample-4.component.scss [hunk 3]
```diff
@@ -1,21 +1,4 @@
...
-    font-size: 1.5em;
-    padding: 16px;
-    text-shadow: 0 1px rgba(0, 0, 0, 0.1);
-    transition: all .35s cubic-bezier(0.0, 0.0, 0.2, 1);
-    cursor: pointer;
-    user-select: none;
-    box-shadow: 0 1px 5px 0 rgba(0, 0, 0, 0.2), 0 2px 2px 0 rgba(0, 0, 0, 0.14), 0 3px 1px -2px rgba(0, 0, 0, 0.12);
+.sample-button{
+    width: 7rem;
+    margin: 10px;
```
Comment (simeonoff) [inline src/app/interactions/ripple/ripple-sample-4/ripple-sample-4.component.scss]:
```suggestion
    margin: 8px;
```

File: src/app/interactions/ripple/ripple-sample-5/ripple-sample-5.component.scss [hunk 4]
```diff
@@ -1,3 +1,4 @@
 .sample-button{
     width: 7rem;
-}
\ No newline at end of file
+    margin: 10px;
```
Comment (simeonoff) [inline src/app/interactions/ripple/ripple-sample-5/ripple-sample-5.component.scss]:
```suggestion
    margin: 8px;
```

File: src/app/interactions/ripple/ripple-sample-6/ripple-sample-6.component.scss [hunk 5]
```diff
@@ -1,3 +1,4 @@
 .sample-button{
     width: 7rem;
-}
\ No newline at end of file
+    margin: 10px;
```
Comment (simeonoff) [inline src/app/interactions/ripple/ripple-sample-6/ripple-sample-6.component.scss]:
```suggestion
    margin: 8px;
```

File: src/app/interactions/ripple/ripple-stying/ripple-styling-sample.component.html [hunk 6]
```diff
@@ -1,92 +1,27 @@
...
-            </article>
-        </section>
-    
-    </div>
\ No newline at end of file
+    <section class="section">
+        <article class="sample-column">
+            <h5>Button</h5>
+            <div class="button-sample">
+                <button class="sample-button" igxButton="outlined" igxButtonColor="#217346" igxButtonBackground="white"
```
Comment (simeonoff) [inline src/app/interactions/ripple/ripple-stying/ripple-styling-sample.component.html]:
```suggestion
                <button class="sample-button" igxButton="outlined"
```

File: src/app/interactions/ripple/ripple-stying/ripple-styling-sample.component.html [hunk 7]
```diff
@@ -1,92 +1,27 @@
...
-                <h5 class="sample-title">Solid Background</h5>
-                <div class="ripple-sample dark" igxRipple>
-                    <p style="margin-bottom:0">Click me to see my ripple!</p>
-                </div>
-            </article>
-        </section>
-    
-    </div>
\ No newline at end of file
+    <section class="section">
```
Comment (simeonoff) [inline src/app/interactions/ripple/ripple-stying/ripple-styling-sample.component.html]: Do we need these wrapping elements?

File: src/app/interactions/ripple/ripple-stying/ripple-styling-sample.component.html [hunk 8]
```diff
@@ -1,92 +1,27 @@
...
+            <div class="button-sample">
+                <button class="sample-button" igxButton="outlined" igxButtonColor="#217346" igxButtonBackground="white"
+                    igxRipple>Click Me
+                </button>
+            </div>
+        </article>
+        <article class="sample-column">
+            <h5>Icon</h5>
+            <div class="button-sample">
+                <button class="shadow" igxButton="icon" igxButtonColor="#217346" igxButtonBackground="white" igxRipple>
```
Comment (simeonoff) [inline src/app/interactions/ripple/ripple-stying/ripple-styling-sample.component.html]:
```suggestion
                <button class="shadow" igxButton="icon" igxRipple>
```

File: src/app/interactions/ripple/ripple-stying/ripple-styling-sample.component.html [hunk 9]
```diff
@@ -1,92 +1,27 @@
...
+                    <igx-icon fontSet="material">search</igx-icon>
+                </button>
+            </div>
+        </article>
+        <article class="sample-column">
+            <h5>Avatar</h5>
+            <div class="button-sample">
+                <igx-avatar class="shadow" bgColor="#72da67" color="white" igxRipple  icon="person" roundShape="true" size="small">
```
Comment (simeonoff) [inline src/app/interactions/ripple/ripple-stying/ripple-styling-sample.component.html]:
```suggestion
                <igx-avatar class="shadow" igxRipple  icon="person" roundShape="true" size="small">
```

File: src/app/interactions/ripple/ripple-stying/ripple-styling-sample.component.html (hunk 7 above)
Comment (SisIvanova) [reply src/app/interactions/ripple/ripple-stying/ripple-styling-sample.component.html]: We need most of them in order to display the three examples in one line. I tried to improve the sample.

**Pull request merged**